src/lib/output/renderer.ts

```diff
@@ -206,14 +206,12 @@ export class Renderer extends ChildableComponent<Application, RendererComponent>
                 this.theme = this.addComponent('theme', new DefaultTheme(this, path));
             } else {
                 try {
-                    const themeClass = typeof require(filename) === 'function' ? require(filename) : require(filename).default;
+                    const themeClass = typeof require(filename) === 'function' ? require(filename)() : require(filename).default;
```

Comment:
Can you please share the code or a description on exactly what you're trying to do with this? I can't see how it relates to the error you're describing. The error you're describing is usually caused by passing `undefined` to a path function.